_data/istio.yml

```diff
@@ -1 +1,2 @@
 version: 0.3.0
+latest_blog_post: 2017/mixer-spof-myth.html
```

Comment:
anyway to not need this to be manually updated / have it be the latest automatically?

Reply:
I don't know how, given our current workflow. I at least managed to centralize things so there's only one place to update.

Maybe there'd be a way to look at the collation order on the blog post page and always target the top-most page. I'll give that a whirl.